app/views/request/_sidebar.html.erb

```diff
@@ -1,49 +1,66 @@
 <div id="right_column" class="sidebar right_column" role="complementary">
-  <div class="new-request-cta">
+  <div class="sidebar__section new-request-cta">
     <%= render :partial => "general/new_request" %>
   </div>
 
+  <% if show_pro_upsell?(@user) %>
+    <div class="sidebar__section pro-upsell">
+      <% pro_link = link_to account_request_index_path do %>
+        <%= AlaveteliConfiguration.pro_site_name -%>
+      <% end %>
+
+      <% pro_site_name_link = content_tag(:strong, pro_link) %>
+
+      <%= _('{{pro_site_name_link}} is a powerful, fully-featured FOI ' \
```

Comment:
hmm, thinking in terms of translator effort and release blockers, this is the same wording as the account request page but a different variable name

https://github.com/mysociety/alaveteli/blob/develop/app/views/alaveteli_pro/account_request/index.html.erb#L4-L6

Reply:
Fixed in c62120410